src/main/java/Gui.java

```diff
@@ -125,6 +125,11 @@ public void actionPerformed(ActionEvent event) {
                     Logger.println(" => Excel files read.");
                     Logger.println("Copying content from FROM excel file to TO files ...");
                     new Consolidator(Properties.getMappingFilePath(), fromExcel, toExcels);
+                    if(Properties.abortRequested()) {
+                        Logger.println(" => User requested to abort copy operations.");
+                        Logger.println("Excel2Excel finished without modifying any files.");
+                    	return;
```

Comment:
I've noticed that in a few of the diffs on the previous pull request, indentation looks fine to me in the resulting file, it's only off in the diff